tangelo/linq/target/target_cirq.py

```diff
@@ -73,7 +77,10 @@ def simulate_circuit(self, source_circuit: Circuit, return_statevector=False, in
         cirq_initial_statevector = initial_statevector if initial_statevector is not None else 0
 
         # Calculate final density matrix and sample from that for noisy simulation or simulating mixed states
-        if self._noise_model or source_circuit.is_mixed_state:
+        # Calculate final density matrix and sample from that for noisy simulation or simulating non-saved mixed states
```

Comment:
I think L79 or L80 needs to be removed or moved.

Reply:
ok